test/integration/test_datatype_upload_irods.py

```diff
@@ -0,0 +1,40 @@
+import os
+import tempfile
```

Comment:
```suggestion
```

Reply:
Can you push the tempfile fixture to a conftest.py file please ? https://docs.pytest.org/en/latest/fixture.html#conftest-py-sharing-fixture-functions

Reply:
Done.